doc/docs/openapi.md

```diff
@@ -130,6 +130,44 @@ import sttp.apispec.openapi.circe._
 println(Printer.spaces2.print(docs.asJson))
 ```
 
+### Support for OpenAPI 3.1.0
+
+Generating OpenAPI documentation compatible with 3.1.0 specifications is a matter of using a different encoder.
+For example, generating the OpenAPI 3.1.0 YAML string can be achieved by performing the following steps:
+
+Firstly add dependencies:
+```scala
+"com.softwaremill.sttp.tapir" %% "tapir-openapi-docs" % "@VERSION@"
```

Comment:
I've noticed, that in lines `16` and `58` instead of `"@VERSION@"` the `"1.1.2"` string is used, is this incorrect and should be updated, or am I missing something?

Reply:
Two alternatives:
1. if it started working with `1.1.2` I would changed it in those lines to `[1.1.2)` (to mark that the minimal version is `1.1.2` but it will work with newer versions too
2. keep `@VERSION` if the intention is to use the latest versions

Reply:
yeah we probably should use `@VERSION` everwhere, but I think both are substituted on release

Reply:
I'm updating explicit `"1.1.2"` to `"@VERSION@"` then. Thanks!